documentation/host/data.rst

```diff
@@ -31,7 +31,7 @@ On Unix:
 
 .. code-block:: bash
 
-   $ sudo apt-get install postgresql-12
+   $ sudo apt-get install postgresql-12  # replace 12 with the version available in your packages
```

Comment:
I think here we should make clear that the version needs to be [>9](https://flexmeasures.readthedocs.io/en/latest/host/data.html) and <max_supported_version.

Reply:
I don't think we have a Max version....